synthnn/core/musical_synthesis.py

```
@@ -426,13 +427,15 @@ class MusicalResonantNetwork(ResonantNetwork):
     Musical extension of ResonantNetwork with synthesis capabilities.
     """
     
-    def __init__(self, name: str = "musical_network", 
+    def __init__(self, name: str = "musical_network",
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)